src/index.js

```diff
@@ -1 +1,141 @@
-import './style.css';
\ No newline at end of file
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Hello, I would suggest that you should not disable linter because it is the tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Can you fix the issue without deleting the linter?
- [ ] **NB: Run your App to make sure it is working before submitting it for re-review**

Reply:
It did not work any other way ? That was the last resort

Reply:
- **You modified the linter's file and you pushed it to GitHub while your code was under review, why did you do that? It is prohibited to push any code to Github when your work is already submitted and under review. It creates confusion. Now the Pull request shows that you submitted a project without Linters. I would advise you not to repeat this behavior.**